src/web/services/student.service.ts

```diff
@@ -124,4 +127,14 @@ export class StudentService {
     const responseType: string = 'text';
     return this.httpRequestService.get(ResourceEndpoints.STUDENTS_CSV, paramsMap, responseType);
   }
+
+  /**
+   * Returns a list of students enrolled in a course.
+   */
+  getStudentsEnrolledInCourse(queryParams: { courseId: string }): Observable<StudentListResults> {
```

Comment:
Why not merge with `getStudentsFromCourse`?

Reply:
I tried haha, as I mentioned in my comment the `StudentListResults` is not of `Students` type because of the difference in the way they refer to "Teams" and "Sections". I could try to standardise them but that would require extensive code rewrite in a lot of other components, might be worth doing in a separate issue?

Reply:
Let's do it in this PR. Even if you do not want to do it in this PR. We still need to address this: https://github.com/TEAMMATES/teammates/pull/10095#discussion_r429117412

Here is the step:
1. Delete type `StudentListResults`
1. Delete type `StudentAttributes` and use `Student` instead
1. Fix code that turns to read.

I hope you can see why we want to do this (instead of just creating a mapping from the data returned by API endpoint to the `StudentListResults`)

Reply:
I tried this at first but I encountered problems with the handsontable mapping (elaborated on in https://github.com/TEAMMATES/teammates/pull/10095#issuecomment-632532308) but I googled a fix and looks like everything is working, manually verified :)